label_studio/tests/sdk/test_tasks.py

```diff
@@ -6,6 +6,7 @@
 
 pytestmark = pytest.mark.django_db
 from label_studio_sdk import Client
+
```

Comment:
let's revert the changes here and below in the diff view, they are causing ruff to fail

Reply:
idk why did these changes appear, since I haven't touch it manually but only with `make fmt-all`

Reply:
What directory are you running `make fmt-all` from? Ruff uses the following section order for dependencies:

https://docs.astral.sh/ruff/settings/#lint_isort_section-order

And I wonder if it thinks `tests` is a third-party dependency because there is no `tests` folder in the directory `ruff` is being executed from

Reply:
Cool, thanks, it is actually the opposite: another `tests` folder is in the same root directory where I executed the script, so it obfuscated the third-party